benchmark-service/benchmark/templates/base.html

```diff
@@ -1,11 +1,13 @@
 <!DOCTYPE html>
 <html lang='en'>
 <head>
-    <meta charset='UTF-8'>
-    <title>{% block title %}
-        Index
-        {% endblock %}
-    </title>
+    <title>Hail | {% block title %}{% endblock %}</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1"></meta>
+    <link rel="shortcut icon" href="{{ base_path }}/common_static/hail_logo_sq.ico" type="image/x-icon" />
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
Why did you have to add these? I think you can inherit from our existing jinja templates with this:

```
{% extends "layout.html" %}
```

An example is in `batch/batch/front_end/templates/batch.html`